CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.4.2 (2021-02-11)
+
+- Fixed the toolbar component to have proper visual styles when built for production. [#190](https://github.com/blackbaud/skyux-layout/pull/190)
```

Comment:
```suggestion
- Fixed the toolbar component's visual styles. [#190](https://github.com/blackbaud/skyux-layout/pull/190)
```
No big deal if you don't want to change this, but the wording felt a little clunky to me.

Reply:
I think I would rather leave this as is. I agree it is a little clunky; however, this issue only affected things in production and didn't really involve changing any styles. I think the clunky version more accurately reflects what the release changed.

Reply:
Sounds good. Thanks!